lib/omnibus/software.rb

```diff
@@ -741,7 +741,7 @@ def with_standard_compiler_flags(env = {}, opts = {})
           elsif platform_version.satisfies?(">= 5.11")
             solaris_flags = {
               "CC" => "gcc -m64 -static-libgcc",
-              "LDFLAGS" => "-Wl,-rpath,#{install_dir}/embedded/lib -L#{install_dir}/embedded/lib -static-libgcc",
```

Comment:
Why modify solaris when we don't for example touch a few other unices that we don't really support?

Reply:
👍